packages/twenty-front/src/modules/object-record/record-group/states/selectors/visibleRecordGroupIdsComponentFamilySelector.ts

```diff
@@ -0,0 +1,84 @@
+import { recordGroupDefinitionFamilyState } from '@/object-record/record-group/states/recordGroupDefinitionFamilyState';
+import { recordGroupIdsComponentState } from '@/object-record/record-group/states/recordGroupIdsComponentState';
+import { RecordGroupDefinition } from '@/object-record/record-group/types/RecordGroupDefinition';
+import { RecordGroupSort } from '@/object-record/record-group/types/RecordGroupSort';
+import { recordGroupSortedInsert } from '@/object-record/record-group/utils/recordGroupSortedInsert';
+import { recordIndexRecordGroupHideComponentFamilyState } from '@/object-record/record-index/states/recordIndexRecordGroupHideComponentFamilyState';
+import { recordIndexRecordGroupSortComponentState } from '@/object-record/record-index/states/recordIndexRecordGroupSortComponentState';
+import { recordIndexRecordIdsByGroupComponentFamilyState } from '@/object-record/record-index/states/recordIndexRecordIdsByGroupComponentFamilyState';
+import { createComponentFamilySelectorV2 } from '@/ui/utilities/state/component-state/utils/createComponentFamilySelectorV2';
+
+import { ViewComponentInstanceContext } from '@/views/states/contexts/ViewComponentInstanceContext';
+import { ViewType } from '@/views/types/ViewType';
+import { isDefined } from '~/utils/isDefined';
+
+export const visibleRecordGroupIdsComponentFamilySelector =
```

Comment:
While testing this I encountered this bug 

https://github.com/user-attachments/assets/dcb94d30-2caf-4990-b820-3800007b17b2

Reply:
After lot of investigation, it seems related to this part:

```typescript
        update: (cache, { data }) => {
          const record = data?.[mutationResponseField];

          if (!record || !cachedRecord) return;

          triggerUpdateRecordOptimisticEffect({
            cache,
            objectMetadataItem,
            currentRecord: cachedRecord,
            updatedRecord: record,
            objectMetadataItems,
          });
        },
```

Inside `useUpdateOneRecord`, feels like we can remove it because we're already updating the cache above ?
Do you agree @charlesBochet, @lucasbordeau ?
It seems to be implemented in other hook that way to, so maybe, we'll have similar issue in other hooks.